microsetta_interface/templates/new_participant.jinja2

```diff
@@ -118,6 +118,11 @@
   width: 80%;
   margin: 0 auto;
 }
+
+.required-field {
```

Comment:
Looks like a bunch of the spans use class="red right required-field", I'm guessing class red just sets color: red.  Should we bundle the definition of right into here also, then we can just set class="required-field" everywhere and drop the red and right classes from all the spans?

Reply:
The "red" and "right" classes were already in the code but didn't appear to be doing anything. I added the required-field class to handle the aesthetic changes I was making, but left red and right in place in case there was some legacy reason for them that I was unaware of.

If there's no reason to keep them, I'd be happy to clean them all out and just keep required-field.

Reply:
Ahh, so spans are required-field, divs are red right required-field?  I think my question is can we drop the red class from the divs (assuming the right class does something else we want that isn't covered)?

Reply:
As far as I can tell, both the red and right classes can be dropped from the divs. Neither class is defined anywhere in the CSS and I don't see any JavaScript that uses them. If neither you nor @wasade know of any reasons to keep them, I can drop both and just leave required-field in place.

Reply:
I'd say drop them then, if nothing shifts around or changes color, we're in the clear

Reply:
I just dropped all of the references to the red and right classes and standardized all of the asterisks to <span class="required-field">. I double-checked after the change and nothing was altered visually, either in color or positioning.